packages/admin/cms-admin/src/blocks/image/EditImageDialog.tsx

```diff
@@ -46,6 +51,7 @@ interface Props {
         height: number;
         size?: number;
     };
+    damFileId?: string;
```

Comment:
```suggestion
    damFileId: string;
```

This can't be undefined